shared/teams/channel/header.tsx

```diff
@@ -148,8 +157,8 @@ const HeaderTitle = (props: HeaderTitleProps) => {
   )
 
   const tip = (
-    <Kb.Box2 direction="horizontal" alignSelf="flex-start" gap="tiny">
-      <Kb.Icon color={Styles.globalColors.black_20} fontSize={12} type="iconfont-info" />
+    <Kb.Box2 direction="horizontal" alignSelf="flex-start" gap="tiny" style={styles.tipBox}>
+      <Kb.Icon color={Styles.globalColors.black_20} type="iconfont-info" sizeType="Small" />
       <Kb.Text type="BodySmall" lineClamp={3}>
```

Comment:
On narrower window widths this text ellipsizes. Should it?

Reply:
It never gets that small, but I'll drop the clamp. 👍